docs/articles/clustering.md

```diff
@@ -56,25 +59,25 @@ zot maintains two types of durable state:
 -   the image metadata in the registry’s cache
 
 In a stateless clustering scheme, the image data is stored in the remote
-storage backend and the registry cache is disabled by turning off both
-deduplication and garbage collection.
+storage backend and the registry cache is disabled by turning off
+deduplication.
 
 ## Ecosystem tools
 
 The [OCI Distribution
 Specification](https://github.com/opencontainers/distribution-spec)
 imposes certain rules about the HTTP URI paths to which various
 ecosystem tools must conform. Consider these rules when setting the HTTP
-prefixes during loadbalancing and ingress gateway configuration.
+prefixes during load balancing and ingress gateway configuration.
 
 ## Examples
 
-zot supports clustering by using multiple stateless zot replicas with shared S3 storage and an `HAProxy` (with sticky session) load-balancing traffic to the replicas.
+Clustering is supported by using multiple stateless zot replicas with shared S3 storage and an HAProxy (with sticky session) load balancing traffic to the replicas. Each replica is responsible for one or more repositories.
 
-### YAML configuration
+### HAProxy YAML configuration
```

Comment:
I'm not sure if this config is actually YAML.

As far as I am aware, haproxy uses a custom config file format as mentioned here: https://www.haproxy.com/documentation/haproxy-configuration-manual/latest/#2.1

Reply:
Changed to HAProxy configuration